apps/acrs/sections/summary-data-sections.js

```diff
@@ -1,7 +1,82 @@
 'use strict';
+const moment = require('moment');
+const PRETTY_DATE_FORMAT = 'DD MMMM YYYY';
 
 module.exports = {
-  start: [
-    'name'
-  ]
+  'referral-details': {
+    steps: [
+      {
+        field: 'brp',
+        omitChangeLink: true
+      },
+      {
+        field: 'uan',
+        omitChangeLink: true
+      },
+      {
+        field: 'date-of-birth',
+        parse: date => date && moment(date).format(PRETTY_DATE_FORMAT),
+        omitChangeLink: true
+      },
+      {
+        steps: '/who-is-completing-form',
+        field: 'who-is-completing-form'
+      },
+      {
+        steps: '/full-name',
+        field: 'full-name'
+      },
+      {
+        steps: '/confirm-referrer-email',
+        field: ''
+      }
+    ]
+  },
+  'immigration-adviser-details': {
+    steps: [
+      {
+        field: 'legal-representative-fullname',
+        parse: (list, req) => {
+          if (!req.sessionModel.get('steps').includes('/immigration-adviser-details')) {
+            return null;
+          }
+          return list;
+        },
+        omitChangeLink: true
```

Comment:
In the final summary the change link is present for the immigration-advisor-details fields. Might want to check that one with the UCD team